File: jscomp/test/demo_pipe.js
'use strict';


function register(rl) do
  return rl.on("line", (function (x) do
                  console.log(x);
                  return --[ () ]--0;
                end)).on("close", (function (param) do
                console.log("finished");
                return --[ () ]--0;
              end));
end

exports.register = register;
--[ No side effect ]--

File: jscomp/test/demo_pipe.lua
__console = {log = print};


function register(rl) do
  return rl.on("line", (function(x) do
                  __console.log(x);
                  return --[[ () ]]0;
                end end)).on("close", (function(param) do
                __console.log("finished");
                return --[[ () ]]0;
              end end));
end end

exports = {};
exports.register = register;
return exports;
--[[ No side effect ]]
